test/utils/Fuzzers.sol

```diff
@@ -87,12 +87,10 @@ abstract contract Fuzzers is Constants, Utils {
         uint128 estimatedDepositAmount = segments[0].amount;
 
         // Fuzz the other segment amounts by bounding from 0.
-        unchecked {
```

Comment:
this shouldn't have been removed

Reply:
Thanks for the comment, Andrei. I considered not removing it but I couldn't understand why we have `unchecked` in this place. This is the code required for testing so what is the rationale behind optimising for gas in here?

Reply:
i remember making the same comment back when this file was initially written. @PaulRBerg, you pointed out that using the `unchecked` block would also slightly decrease compilation time. do i remember that correctly, Paul? and is that still the case?

Reply:
it's not about compilation times - but about run times.

Looping over fuzzed arrays is a computationally heavy operation. I haven't benchmarked this, but I'm pretty sure that the `unchecked` blocks help in this circumstance - it's basically ~100 gas saved per loop, which can add to a quite high consumption for large fuzzing campaigns (e.g., in the CI Deep workflow).

Reply:
Got it. Thanks for explaining Paul. I will revert them now.